newIDE/app/src/AssetStore/InstallAsset.js

```diff
@@ -419,6 +419,14 @@ export const addSerializedExtensionsToProject = (
       'unserializeFrom',
       project
     );
+
+    if (serializedExtension.url) {
+      // extensions with url field are extensions that come from the store.
+      newEventsFunctionsExtension.setOrigin(
+        'extensions-store',
```

Comment:
nit: We could even go for 'gdevelop-extensions-store' as we used 'gdevelop-assets-store' I think for resources

Reply:
👍